### DISCUSSION TITLE
Deploy Keys not showing

### DISCUSSION THREAD

**Discussion by scandefi** [community/community#9989] [Projects and Issues]
Hi there,

I'm trying to allow write access to a deploy key, but when I try to add a new key, an error is occurring (Key already in use) but also I'm not seeing any key into Deploy Keys section with message  "There are no deploy keys for this repository Check out our guide on deploy keys to learn more."

**Comment by gabrielmachts** [comment]
I have the exact same issue.
Also, I can't pull from the repo with said deploy key (via Plesk). I get this message from Plesk:

...
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ 
@ WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED! @ 
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ 
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY! 
Someone could be eavesdropping on you right now (man-in-the-middle attack)! 
It is also possible that a host key has just been changed. 
The fingerprint for the RSA key sent by the remote host is .... 
Please contact your system administrator. 
Add correct host key in /.../git_known_hosts to get rid of this message. 
Offending RSA key in /.../git_known_hosts:1 
RSA host key for github.com has changed and you have requested strict checking. 
Host key verification failed. 
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.

**Reply by gabrielmachts** [reply]
Okay, not being able to pull and the warning message was obviously related to this article: https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/
Deleting git_known_hosts on my server resolved that issue.

But the issue of the OP still persists.
